app/app.py

```diff
@@ -304,6 +307,13 @@ async def lifespan(app: FastAPI) -> AsyncGenerator:
     ):
         hyperion_error_logger.info("Redis client not configured")
 
+    password_dict_files = os.listdir("assets/password_dict")
```

Comment:
Could we add a comment explaining why we add these dicts and where they come from?